packages/tools/fluid-runner/README.md

```diff
@@ -5,6 +5,10 @@ This package contains utility for running various functionality inside a Fluid F
 Allows some execution to be made on a container given a provided ODSP snapshot.
 
 ### Sample command
+If package is installed globally:
+`node fluid-runner exportFile --codeLoader=compiledBundle.js --inputFile=inputFileName.fluid --outputFolder=outputFolderName --scenario=test --telemetryFile=telemetryFile.txt`
+
+If working directly on this package, first navigate to the `bin` directory:
```

Comment:
Running from root of directory will be probably more common, and I feel like using this makes it more obvious what happens:

node bin/fluidRunner ...

That said, I use back slashes on windows - forward slashes works as well, but it's more awkward to use (as auto-compete does not work in standard shell)